pyoverkiz/types.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
+import json
+from typing import Any, Callable, Dict, List, Union
+
+from pyoverkiz.enums import DataType
+
+OverkizStateType = Union[str, int, float, bool, Dict[Any, Any], List[Any], None]
```

Comment:
```suggestion
OVERKIZ_STATE_TYPE = Union[str, int, float, bool, Dict[Any, Any], List[Any], None]
```

Reply:
It seems that for types (so no official constant?), this is not enforced. At least not in Home Assistant. https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/typing.py

Perhaps we should just call it StateType and import it as OverkizStateType when we use it in Home Assistant. Would that be okay?

Reply:
It’s ok for me.